roles/oidcng/templates/application.yml.j2

```diff
@@ -47,38 +50,21 @@ features:
   # Do we enforce that the scopes granted to an access_token are configured with the RS?
   enforce-scope-resource-server: false
 
+sp:
+  entity_id: {{ oidcng_saml_sp_entityid }}
+  acs_location: https://{{ oidcng_base_hostname }}/login/saml2/sso/oidcng
+
 idp:
-  alias: engine-{{ env }}
-  # comma separated string of metadata url's
-  metadata_urls: {{ oidcng_idp_metadata_url }}
-  name_id: https://engine.{{ base_domain }}/authentication/idp/metadata
+  entity_id: https://engine.{{ base_domain }}/authentication/idp/metadata
+  sso_location: https://engine.{{ base_domain }}/authentication/idp/single-sign-on
   metadata_signing_certificate_path: file://{{ oidcng_dir }}/SURFconext-metadata-signer.pem
```

Comment:
I assume that this is the SAML assertion signing key? It might make sense to rename this parameter to reflect that

Reply:
Done